File: Random Number Generator/somkuwaryash/random.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>
int rand_interval(unsigned int min, unsigned int max)
{
    srand(time(NULL));
    return (rand() % (max + 1 - min)) + min;
}

int main()
{
    int i, j;
    char buf;
    printf("Generate a Random number between: i and j\n");
    printf("i: ");

    scanf("%d%c", &i, &buf);
    printf("j: ");
    scanf("%d", &j);

    int x = rand_interval(i, j);
    printf("%d\n", x);
}

File: Random Number Generator/thecodeboy/random.go
package main

import (
	"fmt"
	"math/rand"
	"time"
)

// returns a random number between min and max
func randomInt(min, max int) int {
	rand.Seed(time.Now().UnixNano())
	return min + rand.Intn(max-min)
}

func main() {
	fmt.Println(randomInt(5, 20))
	fmt.Println(randomInt(0, 100))
	fmt.Println(randomInt(-50, -20))
	fmt.Println(randomInt(1000, 2000))
}
